### PR TITLE
Linking the carousel topic titles out to their respective pages

### LINKED ISSUE #648: Turn topic titles into links on the home page topic carousels

### PR DESCRIPTION
**Pull request by DerekSgov**
## Summary
-Linking the Topic header (on the home page) and the carousel topic titles to their respective pages

## Notable Changes 
-Changed the "Hover:" CSS for the Topic header on the home page
-Added link in CollectionPreview.js for the "Collection title"
-Updated snapshot

Close #648

### CODE REVIEW

File: app-web/src/components/Home/index.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ export const StyledLink = styled(Link)`
   text-decoration: none;
   :hover {
     color: inherit;
-    text-decoration: none;
+    text-decoration: underline;
+    text-decoration-color: #1a5a96;
```
Comment (patricksimonian) [inline app-web/src/components/Home/index.js]: is this color not happening by default?

File: app-web/src/components/CollectionPreview/CollectionPreview.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
+import { Link } from 'gatsby';
```
Comment (patricksimonian) [inline app-web/src/components/CollectionPreview/CollectionPreview.js]: Prefer importing our homemade Link component which wraps this component
`import { Link } from '../../UI/Link';`

File: app-web/src/components/Home/index.js (hunk 1 above)
Comment (DerekSgov) [reply app-web/src/components/Home/index.js]: it wasn't no, that was for the "Topics" header

File: app-web/src/components/CollectionPreview/CollectionPreview.js (hunk 2 above)
Comment (DerekSgov) [reply app-web/src/components/CollectionPreview/CollectionPreview.js]: For sure, will update that now

Comment (DerekSgov) [reply app-web/src/components/CollectionPreview/CollectionPreview.js]: updated it on the new push, then styled it a bit for our needs - let me know what you think

**Pull request merged**